addons/web/static/src/legacy/scss/base_frontend.scss

```diff
@@ -24,6 +24,6 @@ html, body, #wrapwrap {
         flex: 1 0 auto;
     }
 }
-.modal-open #wrapwrap {
+.modal-open #wrapwrap:not(.s_has_popup) {
```

Comment:
@adr-odoo This rule will soon be gone so it is kinda annoying... why do we need this? If there is no popup, there no modal-open class normally... right? Why do we need a double class? :grimacing: